src/ripple/core/JobQueue.h

```diff
@@ -237,41 +242,47 @@ class JobQueue : private Workers::Callback
     bool
     isStopped() const;
 
+    /** Returns the number of threads that this job queue is configured with */
+    int
+    getThreadCount() const
+    {
+        return workers_.count();
+    }
+
 private:
     friend class Coro;
 
-    using JobDataMap = std::map<JobType, JobTypeData>;
-
-    beast::Journal m_journal;
-    mutable std::mutex m_mutex;
-    std::uint64_t m_lastJob;
-    std::set<Job> m_jobSet;
+    beast::Journal journal_;
+    mutable std::mutex mutex_;
+    std::uint64_t nextId_ = 0;
+    std::set<Job> jobSet_;
     JobCounter jobCounter_;
     std::atomic_bool stopping_{false};
     std::atomic_bool stopped_{false};
-    JobDataMap m_jobData;
-    JobTypeData m_invalidJobData;
+    std::array<JobTypeData, jobTypes.size()> jobData_;
 
     // The number of jobs currently in processTask()
-    int m_processCount;
+    int activeThreads_ = 0;
```

Comment:
I'm trying to understand the semantic meaning of the variable names like activeThreads_, coroutines and workers. In parallel computing, these terms are used in a loosely inter-changeable fashion. There are a lot of comments about the different methods of the Coro class, but can we give a definition of these terms too? 

My understanding is that: Coroutines (objects of the type Coro) are function pointers which are queued for execution in order of their respective priorities. (I know that this is not an accurate description of Coro class, but something like this would be very helpful).

Also, I couldn't find the usages of activeThreads_ data member. Is it used in TaskQueue::processTask() ?